jubako/src/main/java/com/justeat/jubako/PaginatedContentLoadingStrategy.kt

```diff
@@ -30,19 +31,30 @@ open class PaginatedContentLoadingStrategy(private val pageSize: Int = DEFAULT_P
     override fun load(
         lifecycleOwner: LifecycleOwner,
         data: Jubako.Data,
-        onLoaded: (hasMore: Boolean) -> Boolean
+        callback: (state: PaginatedDataState<ContentDescription<Any>>, hasMore: Boolean) -> Unit
     ) {
         logger.log(TAG, "Load", "onReload: $reset, loading observers: ${loadingData.size}")
-        if (BuildConfig.DEBUG) {
-            TEMP_integrityCheck(data.source, data.destination)
-        }
+//        if (BuildConfig.DEBUG) {
```

Comment:
Needed ? Should it be deleted ?

Reply:
not sure, it was a sanity check and was quite useful I think is worth keeping till this guy has a unit test that can confirm the same thing, still think the design of Jubako needs some work though so maybe after that! :)